### PR TITLE
CORTX-31082: Address Codacy Issues

### PR DESCRIPTION
**Pull request by walterlopatka**
## Description
This addresses some issues flagged by Codacy.

In cluster.py, the most important and significant changes are in breaking up the complex function generate_solution_file().  I also reduced line lengths on a number of other lines that flake8 was complaining about.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds new functionality)
- [ ] Breaking change (bug fix or new feature that breaks existing functionality)
- [ ] Third-party dependency update
- [ ] Documentation additions or improvements
- [x] Code quality improvements to existing code or test additions/updates

## Applicable issues

- This change fixes an issue: CORTX-31082

## How was this tested?
I ran cortx-k8s regression test.
I also ran some manual tests around solution.yaml changes.

## Checklist

- [x] The change is tested and works locally.
- [ ] New or changed settings in the solution YAML are documented clearly in the README.md file.
- [x] All commits are signed off and are in agreement with the [CORTX Community DCO and CLA policy](https://github.com/Seagate/cortx/blob/main/doc/dco_cla.md).

If this change addresses a CORTX Jira issue:

- [x] The title of the PR starts with the issue ID (e.g. `CORTX-XXXXX:`)

### CODE REVIEW

File: test/regression/cluster.py [hunk 1]
```diff
@@ -37,7 +38,8 @@ def __init__(self, solution_file, cluster_file=None, logger=None, logdir=None):
             else:
                 solution_file = os.path.relpath(os.path.join(
                                            os.path.dirname(__file__),
-                                           '../../k8_cortx_cloud/solution.example.yaml'))
+                                           '../../k8_cortx_cloud/'
```
Comment (osowski) [inline test/regression/cluster.py:41]: Is this missing a comma separating the two strings?

Comment (walterlopatka) [reply test/regression/cluster.py:41]: This is ok as is.  When one string follows another they will be concatenated.

**Review by osowski — APPROVED** [review]
lgtm!

**Comment by walterlopatka** [comment]
CORTX-31082: Address Codacy Issues

**Pull request merged**